grakn-test/src/test/java/ai/grakn/test/graql/reasoner/QueryTest.java

```diff
@@ -78,22 +80,23 @@ public void testCopyConstructor(){
 
     @Test
     public void testTwinPattern() {
```

Comment:
Could we get more details on this test? A comment or maybe change the name to be more descriptive.